banco.sql

```diff
@@ -372,11 +372,14 @@ CREATE TABLE `cobrancas` (
   `pdf` varchar(255) DEFAULT NULL,
   `vendas_id` int(11) DEFAULT NULL,
   `os_id` int(11) DEFAULT NULL,
+  `clientes_id` int(11) DEFAULT NULL,
   PRIMARY KEY (`idCobranca`),
   INDEX `fk_cobrancas_os1` (`os_id` ASC),
   CONSTRAINT `fk_cobrancas_os1` FOREIGN KEY (`os_id`) REFERENCES `os` (`idOs`) ON DELETE NO ACTION ON UPDATE NO ACTION,
   INDEX `fk_cobrancas_vendas1` (`vendas_id` ASC),
-  CONSTRAINT `fk_cobrancas_vendas1` FOREIGN KEY (`vendas_id`) REFERENCES `vendas` (`idVendas`) ON DELETE NO ACTION ON UPDATE NO ACTION
+  CONSTRAINT `fk_cobrancas_vendas1` FOREIGN KEY (`vendas_id`) REFERENCES `vendas` (`idVendas`) ON DELETE NO ACTION ON UPDATE NO ACTION,
+  INDEX `fk_cobrancas_clientes1` (`clientes_id` ASC),
+  CONSTRAINT `fk_cobrancas_clientes1` FOREIGN KEY (`clientes_id`) REFERENCES `clientes` (`idClientes`) ON DELETE NO ACTION ON UPDATE NO ACTION
```

Comment:
Qual é o motivo de ter chave estrangeira pra clientes aqui?

Pois já dá pra saber qual o cliente por meio da OS relacionada.

Reply:
Foi uma sugestão do @willph , para poder pegar o cliente direto

Reply:
Foi pra facilitar mas pode ser feito de outra forma.
Tranquilo.

Reply:
Se quiser pode fazer de outra forma @seitbnao

Reply:
@willph , vou deixar como está caso queira mudar, facu deu uma apertada

Reply:
Ok. Mas deu tempo de você finalizar essa PR?
Fora isso

Reply:
> 
> 
> @willph , vou deixar como está caso queira mudar, facu deu uma apertada

Bons estudos. Tem um futuro brilhante lhe esperando.
Aguardaremos o seu retorno!

👍

Reply:
> Ok. Mas deu tempo de você finalizar essa PR?
> Fora isso

então acredito que esteja pronta sim

Reply:
Ok. Vou esperar a aprovação, aí faço as alterações que for encontrando. 

Obrigado pela contribuição.

Reply:
Conseguiria fazer essa solicitação?